src/components/modeler/Modeler.vue

```diff
@@ -708,10 +710,11 @@ export default {
       return shape.component != null;
     },
     setShapeStacking(shape) {
-      if (this.isRendering) {
+      if (this.isRendering || shape === this.previouslyStackedShape) {
```

Comment:
If this is called multiple times for the same shape, due to a bug (possibly related to https://github.com/clientIO/joint/issues/1263), the double click event is not fired. This should also prevent multiple unnecessary calls to `toFront`, though I'm not sure if that had any performance implications.

Reply:
Good catch, Rob

Reply:
It seems `setShapeStacking` needs to be called more than once when dealing with lanes, so I had to add an extra check for that.

I posted an update to the issue I linked above; it seems multiple calls to `toFront` does indeed interfere with the emitting of the double click event.